src/fundus/scraping/crawler.py

```diff
@@ -391,7 +396,7 @@ def constant_delay() -> float:
             else:
                 raise TypeError("param <delay> of <Crawler.__init__>")
 
-        scraper = WebScraper(publisher, self.restrict_sources_to, build_delay())
+        scraper = WebScraper(publisher, self.restrict_sources_to, build_delay(), ignore_robots=self.ignore_robots)
```

Comment:
Unfortunately `Robotspy` does not support the request-rate or crawl-delay directives.

> It does not try to be compliant with commonly accepted directives that are not in the current specs such as request-rate and crawl-delay, but it currently supports sitemaps.

Reply:
As far as I can see, crawl-delay and request-rare are partially implemented. Extending the RobotsParser for crawl-delay was actually quite straight-forward. We would then have a CustomRobotsFileParser. I don't know if we would need it, because it is rarely implemented for all use-agents. But I guess it would be nice compliance wise. What do you think? I have a working draft implementation.

Reply:
Could you open a PR on this one so I can see the draft? Thanks :)

Reply:
I did: #601